src/utils/helpers/browser/__spec__.js

```diff
@@ -29,6 +29,12 @@ describe('Browser', () => {
     });
   });
 
+  describe('isDomAvailable', () => {
+    it('returns the window object', () => {
+      expect(Browser.isDomAvailable()).toBe(true);
```

Comment:
This is only testing a single branch, whereas the code offers 3 conditions - should try and test each (is this not why your coverage is failing?)

Reply:
yes I agree but Browser.getWindow() is a static so I can't stub it their window and document.
So can't really test those branches. Am I missing something?

Reply:
```
spyOn(Browser, 'getWindow').and.returnValue(undefined);
```
should help

Reply:
something along the lines of

```
describe('when window does not exist', () => {
  it('returns false', () => {
    spyOn(Browser, 'getWindow').and.returnValue(undefined);
    expect(Browser.isDomAvailable()).toEqual(false);
  });
});

describe('when document does not exist', () => {
  it('returns false', () => {
    spyOn(Browser, 'getDocument').and.returnValue(undefined);
    expect(Browser.isDomAvailable()).toEqual(false);
  });
});

describe('when document.createElement does not exist', () => {
  it('returns false', () => {
    spyOn(Browser, 'getDocument').and.returnValue({ createElement: undefined });
    expect(Browser.isDomAvailable()).toEqual(false);
  });
});
```

Reply:
Nice. I didn't realise that you could do that with imported objects.

Reply:
Didn't think they were truely static in Javascript. Live, Love and Learn !